### PR TITLE
Rename config Replicas type

### PR DESCRIPTION
**Pull request by khanhtc1202**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/config/number_with_suffix.go [hunk 1]
```diff
@@ -22,20 +22,20 @@ import (
 	"strings"
 )
 
-type Replicas struct {
+type NumberWithSuffix struct {
```
Comment (nghialv) [inline pkg/config/number_with_suffix.go:25]: How about: `UnitSuffixableNumber`? 🤔

Comment (khanhtc1202) [reply pkg/config/number_with_suffix.go:25]: to be honest, it looks too "heavy" to me 😂

**Comment by khanhtc1202** [comment]
Instead of rename the existing Replicas type, we add the new one `Percentage` which more generic and used by all other percentage type required field 👌 
/close

**Pull request closed without merging**